.bazelrc_shared

```diff
@@ -6,24 +6,27 @@ build --javacopt="-source 8 -target 8"
 
 build --strategy=ScalaCompile=worker
 build --worker_max_instances=4
-build --worker_sandboxing
+#build --worker_sandboxing
```

Comment:
TODO: Re-enable this and make sure it works

Reply:
Fixed by undoing [this commit](https://github.com/lucidsoftware/rules_scala/commit/037d7440cf3b192b9979ff2f2c1b2c674b433afc)

I think sandboxing got better in 4 and that is what broke this